presto-jdbc/src/test/java/io/prestosql/jdbc/TestPrestoDriverAuth.java

```diff
@@ -230,6 +230,126 @@ public void testFailedUnknownPublicKey()
         }
     }
 
+    @Test
+    public void testSuccessFullSslVerification()
+            throws Exception
+    {
+        String accessToken = Jwts.builder()
+                .setSubject("test")
+                .setHeaderParam(KEY_ID, "33")
+                .signWith(SignatureAlgorithm.RS256, privateKey33)
+                .compact();
+
+        try (Connection connection = createConnection(ImmutableMap.of("accessToken", accessToken, "SSLVerification", "FULL"))) {
+            try (Statement statement = connection.createStatement()) {
+                assertTrue(statement.execute("SELECT 123"));
+                ResultSet rs = statement.getResultSet();
+                assertTrue(rs.next());
+                assertEquals(rs.getLong(1), 123);
+                assertFalse(rs.next());
+            }
+        }
+    }
+
+    @Test
+    public void testSuccessCaSslVerification()
+            throws Exception
+    {
+        String accessToken = Jwts.builder()
+                .setSubject("test")
+                .setHeaderParam(KEY_ID, "33")
+                .signWith(SignatureAlgorithm.RS256, privateKey33)
+                .compact();
+
+        try (Connection connection = createConnection(ImmutableMap.of("accessToken", accessToken, "SSLVerification", "CA"))) {
+            try (Statement statement = connection.createStatement()) {
+                assertTrue(statement.execute("SELECT 123"));
+                ResultSet rs = statement.getResultSet();
+                assertTrue(rs.next());
+                assertEquals(rs.getLong(1), 123);
+                assertFalse(rs.next());
+            }
+        }
+    }
+
+    // TODO: testSuccessCaSslVerificationMismatchedHostnameValidCA()
+
+    // TODO: testSuccessNoneSslVerificationMismatchedHostname()
+
+    // TODO: testSuccessNoneSslVerificationInvalidCA()
+
+    @Test(expectedExceptions = SQLException.class, expectedExceptionsMessageRegExp = "Connection property 'SSLVerification' is not allowed")
```

Comment:
Please use `org.assertj.core.api.Assertions.assertThatThrownBy;` instead of `expectedExceptions`